frontend/src/metabase/parameters/components/ParameterValueWidget.jsx

```diff
@@ -92,6 +95,21 @@ class ParameterValueWidget extends Component {
       return null;
     }
 
+    const canReset =
```

Comment:
This function would be easier to read with the props destructured.

Reply:
IMO, it's a matter of taste — I am not a big fan of unnecessary repetition. So won't change.

Reply:
Sorry guys, not gonna change that 😄

Reply:
Actually this ended up changed. So you win!